### PR TITLE
Fix URL of link to download playbooks RHV user infra

### PR DESCRIPTION
**Pull request by mburke5678**
https://github.com/openshift/openshift-docs/issues/36617
https://github.com/openshift/okd/issues/875

A link in [Downloading the Ansible playbooks](https://docs.okd.io/latest/installing/installing_rhv/installing-rhv-user-infra.html) in _Installing a cluster on oVirt with user-provisioned infrastructure_ uses a {product-version} variable. In OKD this renders as `4`, causing the link to not work, as it needs a `.x` value (for ex: `4.8`). 

~~Used an `ifdef` to hard-code the version number in the link for OKD. Will need to be manually cherry-picked back.~~ 

Preview OKD: http://file.rdu.redhat.com/~mburke/okd-fix-playbook-link/installing/installing_rhv/installing-rhv-user-infra.html#installation-rhv-downloading-ansible-playbooks_installing-rhv-user-infra 
Preview OCP: https://deploy-preview-36623--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-user-infra.html#installation-rhv-downloading-ansible-playbooks_installing-rhv-user-infra

### CODE REVIEW

**Comment by mburke5678** [comment]
@vrutkovs PTAL

**Review by sandrobonazzola — APPROVED** [review]

File: modules/installation-rhv-downloading-ansible-playbooks.adoc [hunk 1]
```diff
@@ -11,6 +11,18 @@ Download the Ansible playbooks for installing {product-title} version {product-v
 
 . On your installation machine, run the following commands:
 +
+ifdef::openshift-origin[]
+[source,terminal,subs=attributes+]
+----
+$ mkdir playbooks
+$ cd playbooks
+$ curl -s -L -X GET https://api.github.com/repos/openshift/installer/contents/upi/ovirt?ref=release-4.8  |
```
Comment (vrutkovs) [inline modules/installation-rhv-downloading-ansible-playbooks.adoc:19]: This is master, so we need 4.10 I guess? Perhaps we should be using more generic "check `upi/ovirt` folder in github.com/openshift/install and pick a corresponding branch"?

Comment (mburke5678) [reply modules/installation-rhv-downloading-ansible-playbooks.adoc]: @vrutkovs Changed this to https://api.github.com/repos/openshift/installer/contents/upi/ovirt?ref=release-<version> ` with text to go to the openshift installer repo for the version.

**Comment by vrutkovs** [comment]
LGTM!

**Pull request merged**